doc/guides/gps_setup.md

```diff
@@ -0,0 +1,57 @@
+---
+module: [kind=guide] gps_setup
+---
+
+# Setting up GPS
+Some quick definitions for this page.
+```
+GPS host: a computer running the GPS program in host mode
+GPS constellation: a group of GPS hosts
+```
+
+Before GPS can be used in a dimention (e.g. the Nether) a GPS host constellation needs to be set up. Traditionally this is done near the max build height for maximum range, but ender modems can beat normal modes in range even from bedrock.
+
+In order to give the best results, a GPS constallation needs atleast four computers. One can theroetically get away with three for most GPS requests, but four changes that to 100% accuracy when the requester is within all of their ranges, provided that they are placed correctly. More than four GPS hosts per constallation is redundent, but it does not harm.
+
+## Building a GPS constellation
+![An example GPS constellation.](images/gps-constellation-example.png){.big-image}
+
+Assuming that you are using four computers per costallation, you will want each computer that makes up the constallation to be out of plane with atleast one other computer in the constalation. To put this another way, if you can make a straight wall or floor (diagonal also counts as straight, having a corner or curve doesn't) that touches all four computers then you need to move one of the computers so that it's nolonger in contact with the wall/floor.
```

Comment:
Typos on row  19:
"constallation" instead of "constellation" (twice)
"constalation" instead of "constellation"